### DISCUSSION TITLE
Webone on Android 5.

### DISCUSSION THREAD

**Discussion by suzumiya39** [atauenis/webone#120] [Q&A] [answered]
I want to connect internet for windows me.
Also, I running termux on android 5.
But, I can't proot-distro Because they are Stopped Support for android5.
So, Please tell me, way the webone runnable on rootless?
(If there is a substitute for proot-distro, please let me.)

**Comment by suzumiya39** [comment, marked as answer]
solved.
I removed service file and new created a rc script.
https://github.com/suzumiya39/webone-kozaic

So now, we can run on Userland, Termux by deb package and binary.

**Reply by PluMGMK** [reply]
Does this actually allow running without proot, or does it just remove the systemd dependency to allow running in proot-distros on older kernels? I don't think it's currently possible to run anything with .NET without proot…

**Reply by suzumiya39** [reply]
PluMGMK is right.
So I used "github workflow" to build the .deb file.
This way you can avoid problems with .NET build.
